config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -39,13 +40,12 @@ presubmits:
       preset-bazel-scratch-dir: "true"
     spec:
       containers:
-      - image: launcher.gcr.io/google/bazel:0.24.1
+      - image: launcher.gcr.io/google/bazel:0.25.2
         command:
         - hack/bazel.sh
         args:
         - test
         - --config=ci
-        - --config=unit
```

Comment:
what does this do?

Reply:
Skips tests with a lint tag, hence our need for a lint job.

https://github.com/kubernetes/test-infra/blob/297f03c3de37a4545ec02e398d03e0d2ef6e943d/.bazelrc#L15-L17